SUMMARY.md

```diff
@@ -1,50 +0,0 @@
-# Summary
```

Comment:
I wouldn't remove this file, instead I would want to update it as per the new tutorial. Once the work is in good shape, we can update it.

Maybe we can write a small script which does the job.

Reply:
Alright. But where would it be used ? I'm not sure about the purpose of this file.

Reply:
Alright. But where would it be used ? I'm not sure about the purpose of this file.

Reply:
> Alright. But where would it be used ? I'm not sure about the purpose of this file.

Even I'm not really sure, but I would suggest we keep it. It looks good and doesn't really mess up the repo.